PubSub/src/Subscription.php

```diff
@@ -127,6 +133,11 @@ class Subscription
      */
     private $iam;
 
+    /**
+     * @var int
+     */
+    private $eodMaxRetryTime;
```

Comment:
Same comment as above. Let's use exactlyOnceDelivery instead of EOS in comments and in variables.